doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,399 @@
+            :end-before: __lightning_experiment_tracking_mlflow_end__
 
-        metrics = {"loss": loss}
+.. dropdown:: Comet
 
-        # No need to guard this behind the rank check anymore - Ray Train does
-        # this automatically in the returned `wandb` object.
-        wandb.log(metrics)
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_model_dl.py
+        :language: python
 
-        # Also report to Ray Train.
-        train.report(metrics)
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_comet.py
+            :language: python
+            :start-after: __lightning_experiment_tracking_comet_start__
 
+.. dropdown:: TensorBoard
+  
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_model_dl.py
+        :language: python
 
-Using Ray Train logger callbacks
---------------------------------
-Ray Train also exposes logging callbacks that automatically report results to
-experiment tracking services. This will use the results reported via the
-:func:`~ray.train.report` API.
+    .. literalinclude:: ../../../../python/ray/train/examples/experiment_tracking/lightning_exp_tracking_tensorboard.py
+            :language: python
+            :start-after: __lightning_experiment_tracking_tensorboard_start__
+            :end-before: __lightning_experiment_tracking_tensorboard_end__
 
-Logger callbacks provide a simple tracking integration that don't require changes to your
-training code. If you need access to more fine grained APIs, use
-the :ref:`native integrations directly <train-experiment-tracking-native>`.
+Common Errors
+=============
 
-Example: Logging to MLflow and TensorBoard
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Missing Credentials
+-------------------
 
-**Step 1: Install the necessary packages**
+**I have already called `wandb login` cli, but still getting 
+"wandb: ERROR api_key not configured (no-tty). 
+call wandb.login(key=[your_api_key])."**
 
-.. code-block:: bash
+This is probably due to wandb credentials are not set up correctly
+on worker nodes. Make sure that you run ``wandb.login`` 
+or pass ``WANDB_API_KEY`` to each training function. 
+See `"setting up credentials" section <setting-up-credentials>` for more details.
 
-    $ pip install mlflow
-    $ pip install tensorboardX
+Missing Configurations
+----------------------
 
-**Step 2: Run the following training script**
+**"databricks_cli.utils.InvalidConfigurationError: 
+You haven't configured the CLI yet!"**
 
-.. literalinclude:: /../../python/ray/train/examples/mlflow_simple_example.py
-   :language: python
+This is usually caused by running ``databricks configure`` which 
+generates ``~/.databrickscfg`` only on head node. Move this file to a shared
+location or copy it to each node.
+See `"setting up credentials" section <setting-up-credentials>` for more details.
```

Comment:
If you accept my suggestion to rename the section to "set up credentials", then this needs to be updated.